server/util.es6

```diff
@@ -154,4 +156,8 @@ util.translateFromIntakeToMiddleLayer = (input) => {
   return result;
 };
 
+util.validateDateTime = (input) => {
+  return /\d{4}-[01]\d-[0-3]\dT[0-2]\d:[0-5]\d:[0-5]\d([+-][0-2]\d:[0-5]\d|Z)/.test(input) && moment(input, 'YYYY-MM-DDTHH:mm:ssZ').isValid();
```

Comment:
Is this validation now occurring both on the angular and the server? This is maybe a crazy thought, but is there a way to think about isomorphic shared resources?